packages/shared/src/lib/reminderFields.ts

```diff
@@ -27,7 +41,29 @@ export const buildReminderFields = (today: Date = new Date()): ReminderFields =>
     };
 };
 
-export const getReminderFields = (): ReminderFields => {
-    // Add campaign-specific reminders here, e.g. Giving Tuesday
-    return buildReminderFields();
+const givingTuesdayStart = new Date('2023-11-10');
+const givingTuesdayCutOff = new Date('2023-11-27');
+
+const givingTuesdayIsActive = (date: Date): boolean =>
+    date >= givingTuesdayStart && date <= givingTuesdayCutOff;
+
+const newYearsEveStart = new Date('2023-11-28');
+const newYearsEveCutOff = new Date('2023-12-30');
+
+const newYearsEveIsActive = (date: Date): boolean =>
+    date >= newYearsEveStart && date <= newYearsEveCutOff;
+
+export const getReminderFields = (
+    countryCode?: string,
+    date: Date = new Date(),
```

Comment:
Is the date argument included here purely to facilitate the tests? The invocations in the banner/epic files don't include a date argument

Reply:
Yes exactly 👍 (The pattern was already in place for `buildReminderFields`, so I've continued using it).